Doc/data/refcounts.dat

```diff
@@ -1878,6 +2234,26 @@ PyObject_TypeCheck:int:::
 PyObject_TypeCheck:PyObject*:o:0:
 PyObject_TypeCheck:PyTypeObject*:type:0:
 
+PyObject_Vectorcall:PyObject *::+1:
+PyObject_Vectorcall:PyObject *:callable:0:
+PyObject_Vectorcall:PyObject *const *:args:0:
```

Comment:
I prefer to have them. We can think of args as being loosely typed as `PyObject **` and since we are also managing double pointers like that, I think we can also manage those.

Reply:
I'd like to do it in a separate PR. I don't want to change existing lines in this specific PR.